### ISSUE TITLE
No version detected when Project element has attributes

### ISSUE THREAD

**Issue by jmartens** [versionize/versionize#61]
Versionize fails to detect the Version in a Visual Studio 2022 project defined like this:

```
﻿<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
        ...
        <Version>1.0.0</Version>
        ...
     </PropertyGroup>
</Project>
```

**Comment by jmartens** [comment]
I think this XPath expression should work and be compatible with current: `//PropertyGroup/Version`

Above XPath expression will find any `Version` attribute within a `PropertyGroup` element, irrespective of where in the document it is located effectively ignoring the different definitions of the `Project` element.

**Comment by saintedlama** [comment]
Having a PR ready to fix this https://github.com/versionize/versionize/pull/62

**Comment by saintedlama** [comment]
Fixed in 1.12.0
